### PR TITLE
TranslatableExtension: remove unused $isRecursive parameter from ctor

### LINKED ISSUE #16: Nesprávné parametry pro TranslatableSubscriber v novém tagu 1.4.0 KnpLabs/DoctrineBehaviors
Zdravím,

v posledním tagu [1.4.0](https://github.com/KnpLabs/DoctrineBehaviors/releases/tag/1.4.0) se změnila implementace ctoru viz [TranslatableSubscriber 1.4.0](https://github.com/KnpLabs/DoctrineBehaviors/blob/1.4.0/src/ORM/Translatable/TranslatableSubscriber.php#L44) a [TranslatableSubscriber 1.3.2](https://github.com/KnpLabs/DoctrineBehaviors/blob/1.3.2/src/ORM/Translatable/TranslatableSubscriber.php#L44). Odstranil se parametr `$isRecursive` a to je potřeba odstranit i v [TranslatableExtension](https://github.com/Zenify/DoctrineBehaviors/blob/master/src/DI/TranslatableExtension.php#L48).

### PR DESCRIPTION
**Pull request by TomasPilar**
Close #16

### CODE REVIEW

**Comment by enumag** [comment]
Can you change the required version in composer.json accordingly?

Don't worry about failed hhvm tests on travis. That's because of nette/neon.

File: composer.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
 	"require": {
 		"php": ">=5.6",
 		"kdyby/events": "~2.3|~3.0",
-		"knplabs/doctrine-behaviors": "~1.2",
+		"knplabs/doctrine-behaviors": "~1.4.0",
```
Comment (enumag) [inline composer.json]: Just 1.4

**Comment by TomasPilar** [comment]
Required version of KnpLabs/DoctrineBehaviors is updated to ~1.4.0.

**Comment by enumag** [comment]
Good. :-) Squash those changes into a single commit with massage like "Update to KnpLabs/DoctrineBehaviors v1,4".

**Comment by TomasPilar** [comment]
Here it is! :)

**Comment by enumag** [comment]
@TomasPilar Merged, thank you!

**Comment by TomasPilar** [comment]
@enumag Thank you too!

**Pull request merged**